tests/dashboard/test_permissions.py

```diff
@@ -0,0 +1,247 @@
+from __future__ import unicode_literals
+
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import Group
+
+from saleor.userprofile.models import User
+from saleor.dashboard.group.forms import GroupPermissionsForm
+from saleor.dashboard.staff.forms import UserGroupForm
+
+
+def test_superuser_permissions(admin_user):
```

Comment:
I'm not sure that we need such tests. Superuser has all permission by default and this is granted by Django. We don't have test that Django works correctly:)

Reply:
Yea, first test. Checking if this works as i expect and i forget about it. Deleted